src/piadm.sh

```diff
@@ -54,50 +54,65 @@ vecho() {
     fi
 }
 
-declare -a allbootable
-declare -a numbootable
+declare bootfs
+declare -a allbootfs
+declare -a numbootfs
```

Comment:
Is this really an array, or is just `allbootfs`?

Reply:
It is an array of all of the bootable filesystems.  It also is the list of bootable pools.

Reply:
Isn't that `allbootfs`, what about `numbootfs`?

Reply:
Doh!  Will fix.